detections/endpoint/windows_shell_process_from_crushftp.yml

```diff
@@ -0,0 +1,51 @@
+name: Windows Shell Process from CrushFTP
+id: 459628e3-1b00-4e9b-9e5b-7da8961aea35
+version: 1
+date: '2025-04-03'
+author: Michael Haag, Splunk
+status: experimental
+type: TTP
+description: Detects execution of a shell process with CrushFTP as a parent process. May indicate post-exploitation of a vulnerability in CrushFTP such as CVE-2025-31161 authentication bypass. This detection leverages data from Endpoint Detection and Response (EDR) agents, focusing on process creation events where the parent process is crushftpservice.exe.
```

Comment:
can we reword this and why this behavior is questionable? maybe lets consider generating this event synthetically so that it can be validated/tested?